src/pages/saksoversikt/beregning/VisBeregning.tsx

```diff
@@ -47,21 +64,29 @@ const VisBeregning = (props: Props) => {
             <table className="tabell">
                 <thead>
                     <tr>
-                        <th></th>
+                        <th>{intl.formatMessage({ id: 'utbetaling.tabellheader.periode' })}</th>
                         <th>{intl.formatMessage({ id: 'utbetaling.tabellheader.beløp' })}</th>
                         <th>{intl.formatMessage({ id: 'utbetaling.tabellheader.grunnbeløp' })}</th>
                     </tr>
                 </thead>
                 <tbody>
-                    {beregning.månedsberegninger.map((beregning) => (
-                        <tr key={beregning.id}>
-                            <td>{`${intl.formatDate(beregning.fom)} - ${intl.formatDate(beregning.tom)}`}</td>
-                            <td>{beregning.beløp}</td>
-                            <td>{beregning.grunnbeløp}</td>
-                        </tr>
-                    ))}
+                    {gruppertMånedsberegninger.map((gruppe) => {
+                        const head = arr.head(gruppe);
+                        const last = arr.last(gruppe);
+                        if (head._tag === 'None') return;
+                        if (last._tag === 'None') return;
+
+                        return (
+                            <tr key={beregning.id}>
+                                <td>{`${intl.formatDate(head.value.fom)} - ${intl.formatDate(last.value.tom)}`}</td>
+                                <td>{head.value.beløp}</td>
+                                <td>{head.value.grunnbeløp}</td>
+                            </tr>
+                        );
```

Comment:
Vi kan gjøre denne litt mer funksjonell og fin:

```suggestion
                        pipe(
                        	combineOptions(Arr.head(gruppe), Arr.last(gruppe)),
                        	Option.fold(
                        		() => null,
                        		([head, last]) => (
	                        		<tr key={beregning.id}>
	                                <td>{`${intl.formatDate(head.value.fom)} - ${intl.formatDate(last.value.tom)}`}</td>
	                                <td>{head.value.beløp}</td>
	                                <td>{head.value.grunnbeløp}</td>
	                            </tr>
                        		)
                        	)
                        );
```

der `combineOptions` defineres slik i `lib/fp.ts`:
```ts
import * as Apply from 'fp-ts/es6/Apply';
import * as Option from 'fp-ts/es6/Option';

export const combineOptions = Apply.sequenceT(Option.option);
```